Automattic/wp-calypso
# A4A: Add Hosting section reusable component.

This PR introduces a reusable Hosting section for the new Hosting page design.

<img width="1458" alt="Screenshot 2024-07-25 at 2 30 25 AM" src="https://github.com/user-attachments/assets/8f0a74b7-92fe-4b22-b552-3f3a6d2115b4">


Closes https://github.com/Automattic/jetpack-genesis/issues/430

## Proposed Changes
* Update the Hosting Overview layout body to span full width.
* Implement HostingSection component that accepts the following props:
   * heading **(required)**
   * subheading 
   * background: { image: string, color: string, size: string }
   * description
* Create a pre-defined Background that can be used in the HostingSection.

## Why are these changes being made?


*

## Testing Instructions

* Use the A4A live link and go to `/marketplace/hosting`.
* Confirm that the new section is visible.
* You can also test different combinations to test the new component in your local environment.


## Pre-merge Checklist
